cmd/client/main.go

```diff
@@ -24,13 +32,14 @@ func main() {
 	done := make(chan struct{})
 	go func() {
 		defer close(done)
+		var msg Message
 		for {
-			mt, msg, err := conn.ReadMessage()
+			conn.ReadJSON(&msg)
 			if err != nil {
 				logger.Log("err", err)
 				return
 			}
-			logger.Log("type", mt, "from", "server", "msg", string(msg))
+			logger.Log("from", "server", "msgId", msg.MsgId, "from", msg.MsgFrom, "To", msg.MsgTo, "content", msg.Content, "createdAt", msg.CreatedAt)
```

Comment:
we can make this more readable this way:
```suggestion
			logger.Log(
			   "from", "server", 
			   "msgId", msg.MsgId, 
			   "from", msg.MsgFrom, 
			   "to", msg.MsgTo, 
			   "content", msg.Content, 
			   "createdAt", msg.CreatedAt,
			)
```

Reply:
cool